### PR TITLE
Added instruction for jest testing

### PR DESCRIPTION
**Pull request by JonghunYu**
resolve https://github.com/tipsi/tipsi-stripe/issues/104

### CODE REVIEW

**Comment by isnifer** [comment]
@JonghunYu awesome, thank you

File: README.md [hunk 1]
```diff
@@ -698,6 +698,17 @@ To solve this problem please be sure that `Stripe.framework` is added to `Link B
 
 We have fixed this issue, but if you somehow facing this bug again - please, create an issue or a pull request and we will take another look.
 
+#### jest
+To make jest work with tipsi-stripe, you should change `transformIgnorePatterns` in `package.json' file. Please refer to [here](https://facebook.github.io/jest/docs/tutorial-react-native.html#transformignorepatterns-customization)
```
Comment (isnifer) [inline README.md]: please fix the last quote in `package.json`

**Comment by JonghunYu** [comment]
@isnifer Thanks for reviewing. I rebased it and push again.

**Comment by cybergrind** [comment]
@JonghunYu thank you for your contribution!

**Comment by JonghunYu** [comment]
@cybergrind you're welcome. Thanks for the library.

**Pull request merged**